x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/DownsampleActionIT.java

```diff
@@ -107,6 +107,7 @@ public void refreshAbstractions() {
             alias,
             policy
         );
+        updateClusterSettings(client(), Settings.builder().put("indices.lifecycle.poll_interval", "5s").build());
```

Comment:
TODO: Remove this

I only had it for my testing

Reply:
But it can be removed now?

Reply:
Removed in https://github.com/elastic/elasticsearch/pull/94965/commits/b669ad9a3391ee8de91e91b6327df8cb8be98a5a